money_bot/handlers/balance_handlers.py

```diff
@@ -0,0 +1,16 @@
+from aiogram import Dispatcher, types
+
+from money_bot.utils import db_utils
+from money_bot.utils.states import GlobalStates
+from money_bot.utils.strings import BALANCE_MENU_TEXT
+
+
+async def entry_point(message: types.Message):
+    user = await db_utils.get_user_by_id(message.from_user.id)
+    await message.answer(
+        BALANCE_MENU_TEXT["balance_info"].format(user.money, await db_utils.get_invited_count(user.user_id))
+    )
+
+
+def register_all_handlers(dp: Dispatcher):
```

Comment:
Rename register_all_handlers to register_handlers in earn_handlers.py, invite_handlers.py, register_all_handlers.py